.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
     "editor.defaultFormatter": "esbenp.prettier-vscode",
     "typescript.preferences.importModuleSpecifier": "project-relative",
     "editor.codeActionsOnSave": {
-        "source.organizeImports": true
+        "source.organizeImports": "explicit"
```

Comment:
What's this do?

Reply:
It's the same as true but not depricated
![image](https://github.com/arkahna/featureboard-sdks/assets/84910463/eb0bbe5c-9cc3-4a8d-871a-7ba8830b440b)